app/com/lynxanalytics/biggraph/frontend_operations/Operations.scala

```diff
@@ -739,4 +742,31 @@ object PythonUtilities {
     val op = graph_operations.DeriveTablePython(code, outputFields(outputs, api).toList)
     op(op.df, table).result.df
   }
+
+  def deriveHTML(
+      code: String,
+      inputs: Seq[String],
+      project: com.lynxanalytics.biggraph.controllers.ProjectEditor)(
+      implicit manager: MetaGraphManager): Scalar[String] = {
+    val api = Seq("vs", "es", "graph_attributes")
+    // Run the operation.
```

Comment:
This is also mostly shared with `derive`, but I just copy-pasted it instead of abstracting it out.